_news/181.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+id: 181
+title: "Erlang/OTP 28.1 Release"
+lead: "Erlang/OTP 28.1 is the first maintenance patch package for OTP 28, with mostly bug fixes."
+tags: "release, OTP, 28.1"
+date: "2025-09-17"
+author: "Henrik Nord"
+---
+## OTP 28.1
+
+Erlang/OTP 28.1 is the first maintenance patch package for OTP 28, with mostly bug fixes as well as improvements.
+
+
+### Potential incompatibilities:
+
+* The internal `inet_dns_tsig` and `inet_res` modules have been fixed to TSIG
+  verify the correct timestamp.
+  In the process two undocumented error code atoms have been corrected to
+  `notauth` and `notzone` to adhere to the DNS RFCs. Code that relied on the
+  previous incorrect values may have to be corrected.
+
+## HIGHLIGHTS
+
+* A User's Guide to `dbg` is now available in the documentation.
+* Support for quantum crypto signature algorithm ML-DSA (ssl and
+  public_key) and key exchange algorithm ML-KEM (ssl).
+
+* Support for ML-DSA and ML-KEM provided by OpenSSL 3.5 (crypto).
```

Comment:
Should we not say crypto support for ML-DSA and ML-KEM  if built and linked with OpenSSL 3.5